client/src/components/FilesDialog/FilesDialog.test.ts

```diff
@@ -5,15 +5,14 @@ import flushPromises from "flush-promises";
 import { selectionStates } from "@/components/SelectionDialog/selectionStates";
```

Comment:
Please fix the typescript errors in this file. Using any casts is fine, since this is just a test.